Makefile

```diff
@@ -1,22 +1,18 @@
-test_14 ::
-	@echo Running unit tests inside of Go 1.4
-	docker run --rm -v `pwd`:/go/src/github.com/SUSE/zypper-docker zypper-docker-testing-1.4 godep go test -race -v ./...
-	@echo Running climate inside of Go 1.4
-	docker run --rm -v `pwd`:/go/src/github.com/SUSE/zypper-docker zypper-docker-testing-1.4 climate -open=false -threshold=80.0 -errcheck -vet -fmt .
-
-test_15 ::
-	@echo Running unit tests inside of Go 1.5
-	docker run --rm -v `pwd`:/go/src/github.com/SUSE/zypper-docker zypper-docker-testing-1.5 godep go test -race -v ./...
-	@echo Running climate inside of Go 1.5
-	docker run --rm -v `pwd`:/go/src/github.com/SUSE/zypper-docker zypper-docker-testing-1.5 climate -open=false -threshold=80.0 -errcheck -vet -fmt .
-
-test_tip ::
-	@echo Running unit tests inside of tip
-	docker run --rm -v `pwd`:/go/src/github.com/SUSE/zypper-docker zypper-docker-testing-tip godep go test -race -v ./...
-	@echo Running climate inside of tip
-	docker run --rm -v `pwd`:/go/src/github.com/SUSE/zypper-docker zypper-docker-testing-tip climate -open=false -threshold=80.0 -errcheck -vet -fmt .
-
-test :: test_14 test_15 test_tip
+VERSIONS = 1.4 1.5 tip
+
+test ::
+		for version in $(VERSIONS);do \
+			echo Running unit tests inside of Go $${version} ...; \
+			docker run --rm -v `pwd`:/go/src/github.com/SUSE/zypper-docker zypper-docker /bin/bash -c "/opt/test.sh -v=$${version}"; \
```

Comment:
You shouldn't have to specify `/bin/bash -c`, by default docker's entrypoint is `sh -c`. Am I missing something?